src/Microsoft.Dnx.Tooling/List/AssemblyWalker.cs

```diff
@@ -42,7 +42,7 @@ public class AssemblyWalker
             _paths = paths;
         }
 
-        public void Walk(IGraphNode<LibraryDescription> root)
+        public void Walk(IGraphNode<LibraryDependency> root)
```

Comment:
Switch to `LibraryDependency` from `LibraryDescription` because the `RequestedVersion` of `LibraryDescription` doesn't represent the actual version asked in `project.json`